partiql-conformance-test-generator/src/schema.rs

```diff
@@ -0,0 +1,102 @@
+use crate::schema::TestCaseKind::Parse;
+use codegen::{Function, Module, Scope};
+
+/// Conformance test document containing namespaces and/or tests
+pub struct TestDocument {
```

Comment:
Does there need to be a difference between a `TestDocument` and a `Namespace`? Perhaps the "root" level for a file could be a `Namespace` with that file's full path?

Reply:
I think having the distinction between a top-level/root-level test document and a namespace is helpful to have. It makes mapping the schema objects to the codegen objects slightly clearer imo. (i.e. mapping `TestDocument` to a codgen's [scope](https://docs.rs/codegen/latest/codegen/struct.Scope.html) and `Namespace` to codgen's [module](https://docs.rs/codegen/latest/codegen/struct.Module.html)).

Also I think modeling root-level as a `Namespace` could be misused (e.g. including a root-level file as some other root-file's `Namespace`).